targets/mbed/Makefile.mbed

```diff
@@ -46,38 +65,36 @@ jerry:
 	 -DEXTERNAL_CMAKE_C_COMPILER=arm-none-eabi-gcc \
 	 -DEXTERNAL_CMAKE_C_COMPILER_ID=GNU \
 	 -DEXTERNAL_COMPILE_FLAGS="$(EXT_CFLAGS)" \
-	 -DEXTERNAL_MEM_HEAP_SIZE_KB=$(JERRYHEAP)
+	 -DEXTERNAL_MEM_HEAP_SIZE_KB=$(JERRYHEAP) \
+	 -DEXTERNAL_PORT_DIR=$(EXT_PORT_DIR)
 
 	make -C $(INTERM) $(TYPE).external
 	cp `cat $(INTERM)/$(TYPE).external/list` $(OUTPUT)/.
 	cp $(OUTPUT)/lib$(TYPE).jerry-core.a $(COPYTARGET)/libjerrycore.a
 	cp $(OUTPUT)/lib$(TYPE).jerry-libm.lib.a $(COPYTARGET)/libjerrylibm.a
 
-
 js2c:
-	cd targets/mbedk64f; ../tools/js2c.py;
-
+	cd targets/mbed; ../tools/js2c.py;
 
 yotta:
-	cd targets/mbedk64f; \
-	 yotta target frdm-k64f-gcc; \
+	cd targets/mbed; \
+	 yotta target $(YOTTA_TARGET); \
 	 yotta build
 
-
-flash: check_mbed
-	cp targets/mbedk64f/build/frdm-k64f-gcc/source/jerry.bin \
-	 "$(TARGET_DIR)/."
+flash:
+	ifndef TARGET_DIR
```

Comment:
ifndef and related lines shouldn't be tab-indented, as they're not shell commands, but part of the Makefile.